generics/services/kendra.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * name : kendra.js
+ * author : Aman Jung Karki
+ * Date : 26-June-2020
+ * Description : All kendra service related information.
+ */
+
+//dependencies
+
+const request = require('request');
+
+/**
+  * Get downloadable file.
+  * @function
+  * @name gcpDownloadFile
+  * @param {Object} bodyData - body data.
+  * @returns {Array} GCP Downloadable file.
+*/
+
+var gcpDownloadFile = function (bodyData) {
```

Comment:
@aman-tunerlabs - Function can be a const and not var. Also think of a common function here, just the endpoint switches based on selected cloud storage in env. Can be a single function with switch case on cloud storage